pkg/api/types.go

```diff
@@ -451,6 +451,45 @@ type AgentPoolProfile struct {
 	ImageRef                     *ImageReference      `json:"imageReference,omitempty"`
 }
 
+// IsEnabledAcc returns if the AcceleratedNetworkingEnabled is explicitly enabled, or the user-provided default if non explicitly enabled
+func (a *AgentPoolProfile) IsEnabledAcc() bool {
```

Comment:
Not sure if we need an extra field/function `IsEnabledAcc()` : instead, we could set a default for `AcceleratedNetworkingEnabled` in https://github.com/Azure/acs-engine/blob/master/pkg/acsengine/defaults.go#L791 (using the same logic as below)